app/import_functions.go

```diff
@@ -1118,6 +1118,10 @@ func (a *App) importReplies(c *request.Context, data []ReplyImportData, post *mo
 		reply.RootId = post.Id
 		reply.Message = *replyData.Message
 		reply.CreateAt = *replyData.CreateAt
+		if reply.CreateAt < post.CreateAt {
+			mlog.Warn("Reply CreateAt is before parent post CreateAt, setting it to parent post CreateAt", mlog.Int64("reply_create_at", reply.CreateAt), mlog.Int64("parent_create_at", post.CreateAt))
```

Comment:
it would be great to add something to the log so anyone reviewing it can figure out where to look in case they can fix it.

Reply:
The warning message looks something like:

```
{"timestamp":"2022-05-25 21:16:28.027 +05:30","level":"warn","msg":"Reply CreateAt is before parent post CreateAt, setting it to parent post CreateAt","caller":"app/import_functions.go:1122","reply_create_at":1653493588010,"parent_create_at":1653493588020}
```

We don't have a post ID here either, we only actually make the posts 30 lines below on line 1152